e3/electrolyt/run.py

```diff
@@ -100,7 +100,7 @@ def do_test(self):
     def do_checkout(self):
         """Get sources from vcs to sandbox vcs_dir."""
         repo_name = self.data.repo_name
-        if self.data.repo_data is None:
+        if self.data.repo_data is None:  # defensive code
```

Comment:
Just wondering, why not just assert, here, since the change in e3.anod.context should guaranty that this is no longer possible, right?

Reply:
I've remove this code. We do not even need an assert.